File: app/src/main/java/com/comm/util/BeaconActivity.java
package com.comm.util;

import com.alibaba.android.arouter.facade.annotation.Route;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

@Route(path = "/app/BeaconActivity")
public class BeaconActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_beacon);
    }
}

File: app/src/main/java/com/comm/util/BeaconActivity.kt
package com.comm.util

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

//@Route(path = "/app/BeaconActivity")
class BeaconActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_beacon)
        findViewById<View>(R.id.bt_learn_more).setOnClickListener {
            val i = Intent()
            i.action = "com.google.android.payments.standard"
            startActivity(i)
        }
    }
}